src/core/styles/colors/utils.ts

```
@@ -1,26 +1,28 @@
+  const splitMatch = className.split("-");
+  if (splitMatch.length < 3) {
+    throw new Error("Invalid TW class format");
   }
 
-  return color;
+  const color = splitMatch[splitMatch.length - 2];
+  const variant = splitMatch[splitMatch.length - 1];
+  const property = splitMatch.slice(0, splitMatch.length - 1).join("-");
+
+  const numericalVariant = Number(variant.slice(0, -2)) ?? 0;
+  if (isNaN(numericalVariant)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)